runtime/manager.go

```diff
@@ -304,22 +341,18 @@ func (m *Manager) shouldSkip(path string, runtime string) (bool, error) {
 		return false, nil
 	}
 
+	for _, re := range m.skipPaths[runtime] {
+		if re.Value.MatchString(path) {
```

Comment:
should we also check for `re.Skip` ?

Reply:
No, we don't we just check if the pattern matches if it matches we return the value of Skip, for negation (pattern with `!` in front of it ) Skip value will be always false and for other pattern Skip will hold true